content/terms_and_definitions/terms_and_definitions.adoc

```diff
@@ -6,7 +6,113 @@ ifndef::include-only-once[]
 include::{root-path}_config.adoc[]
 endif::[]
 
-TODO: Add terms and definitions
+3D asset::
+The combination of an object's 3D geometry, assigned material properties, and
+specific metadata.
 
-Static road network::
-Collection of connected roads enriched by static objects that do not change during runtime of a simulation.
+3D model::
+A mathematical coordinate-based representation of an object's geometrical
+surface. This excludes physical material properties. For a combination of 3D
+geometry and physical material properties, see 3D asset.
+
+Related terms: 3D asset, 3D environment.
+
+3D environment::
+A 3D representation of a simulated sensor's surroundings. It can consist of
+multiple 3D assets.
+
+Co-simulation::
+Simulation of multiple independent models connected with interfaces. A
+co-simulation platform handles the timing and data transfer between the
+individual models.
+
+Ground truth::
+Defines true static and dynamic states of objects. The true states are either
+determined by the content creator or defined by an omniscient observer, for
+example, a simulation platform.
+
+Machine perception::
+The ability of systems to interpret sensory data and to reliably detect objects
+and other road users relevant to the operation of automated driving. The data is
+used to generate situational awareness of the vehicle environment.
+
+* Source: Klaus Dietmayer, Predicting of Machine Perception for Automated Driving, 2016
+
+Material::
+A collection of material properties belonging to one entity.
+
+Material assignment::
+A mapping of material IDs to parts of 3D models. The material ID is a unique
```

Comment:
A mapping of material IDs to parts of a of 3D models or pixels of textures mapped to these parts.